packages/firestore/src/platform_browser/webchannel_connection.ts

```diff
@@ -248,7 +249,7 @@ export class WebChannelConnection implements Connection {
 
     const url = urlParts.join('');
     log.debug(LOG_TAG, 'Creating WebChannel: ' + url + ' ' + request);
-    // tslint:disable-next-line:no-any Because listen isn't defined on it.
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any, Because listen isn't defined on it.
```

Comment:
So almost everywhere else you moved the prose comment onto a separate line.  I slightly prefer it on the same line (so they stay in-sync and don't get separated), but either way we should try to be consistent I think.

Reply:
I made it 2 lines in some places because I was afraid the line becomes too long that prettier would wrap it, which would break the suppression.
I did an experiment with a very long comment and prettier didn't change it, so I think it's safe to put everything in one line. 
I went ahead to make the comment on the same line as the suppression.